treevalue/tree/tree/graph.py

```diff
@@ -134,6 +134,13 @@ def _new_func(current, parent, current_path, parent_path):
 _GENERIC_S = _GENERIC_N // 3
 
 
+def _custom_repr(x):
+    if isinstance(x, (str,)):
```

Comment:
why `(str, )` here

Reply:
Because maybe some other classes will be added soon. Use simple `str` is also okay, this doesn't matter.